atmo/templates/atmo/detail-cluster.html

```diff
@@ -0,0 +1,92 @@
+{% extends "atmo/base.html" %}
+{% block content %}
+
+<h1>Cluster {{ cluster.identifier }}</h1>
+<p class="lead">Summary:</p>
+<div class="row">
+  <div class="col-md-2"><strong>ID:</strong></div>
+  <div class="col-md-10">{{ cluster.jobflow_id }}</div>
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>Name:</strong></div>
+  <div class="col-md-10">{{ cluster.identifier }}</div>
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>State:</strong></div>
+  <div class="col-md-10">{{ cluster.most_recent_status }}</div>
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>Master public DNS:</strong></div>
+  <div class="col-md-10">
+    {% if cluster.master_address %}
+      {{ cluster.master_address }}
+    {% else %}
+    Please wait, instance details will appear when the master is ready.
+    {% endif %}
+  </div>
+</div>
+{% if cluster.is_active %}
+  <div class="row">
+    <div class="col-md-2"><strong>Terminates at:</strong></div>
+    <div class="col-md-10">{{cluster.end_date}}</div>
+  </div>
+  <p>
+    <form action="/clusters/delete/" method="POST" enctype="multipart/form-data">
+      {% csrf_token %}
+      <input name="cluster" value="{{cluster.id}}" type="hidden">
+      <button type="submit" class="btn btn-sm btn-danger" title="Terminate Spark cluster">
+        <span class="glyphicon glyphicon-stop" aria-hidden="true"></span>
+        Terminate now
+      </button>
+    </form>
+  </p>
+  <h3>Your cluster will be killed within {{ cluster.end_date|timeuntil }}. Be sure to save any work!</h3>
+  <p class="lead">Connecting:</p>
+  {% if not cluster.master_address %}
+    <p>
+      Your cluster is still launching. Please wait and the connection
+      details will appear here when it is ready.
+    </p>
+  {% else %}
+    <p>
+      The default username is <code>hadoop</code>, so you can tunnel your IPython
+      notebook using a command like:
+    </p>
+    <p>
+      <code>ssh -i my-private-key -L 8888:localhost:8888 hadoop@{{ cluster.master_address }}</code>
+    </p>
+    <p>
+      You can save some typing by adding defaults to your <code>~/.ssh/config</code> file:
+    </p>
+    <pre>Host *.compute.amazonaws.com
+  User hadoop
+  LocalForward 8888 localhost:8888
+  Identityfile ~/.ssh/{my-private-key}</pre>
+    To upload an existing notebook, use SCP:
```

Comment:
Let's also add a link here `ssh://hadoop@{{ cluster.master_address }}` here, so once people have added the section to `~/.ssh/config` they can just click on it to launch their terminal